manifests/0.5.1/3scale-community-operator.v0.5.1.clusterserviceversion.yaml

```diff
@@ -166,15 +175,25 @@ spec:
                 - name: OPERATOR_NAME
                   value: threescale-operator
                 - name: BACKEND_IMAGE
-                  value: "quay.io/3scale/3scale28:apisonator-3scale-2.8.0-GA"
+                  value: "quay.io/3scale/3scale28:apisonator-3scale-2.8.1-GA"
                 - name: APICAST_IMAGE
-                  value: "quay.io/3scale/3scale28:apicast-3scale-2.8.0-GA"
+                  value: "quay.io/3scale/3scale28:apicast-3scale-2.8.1-GA"
                 - name: SYSTEM_IMAGE
-                  value: "quay.io/3scale/3scale28:porta-3scale-2.8.0-GA"
+                  value: "quay.io/3scale/3scale28:porta-3scale-2.8.1-GA"
                 - name: ZYNC_IMAGE
-                  value: "quay.io/3scale/3scale28:zync-3scale-2.8.0-GA"
+                  value: "quay.io/3scale/3scale28:zync-3scale-2.8.1-GA"
                 - name: SYSTEM_MEMCACHED_IMAGE
                   value: "memcached:1.5"
+                - name: BACKEND_REDIS_IMAGE
+                  value: centos/redis-32-centos7
+                - name: SYSTEM_REDIS_IMAGE
+                  value: centos/redis-32-centos7
+                - name: SYSTEM_MYSQL_IMAGE
+                  value: centos/mysql-57-centos7
+                - name: SYSTEM_POSTGRESQL_IMAGE
+                  value: centos/postgresql-10-centos7
+                - name: ZYNC_POSTGRESQL_IMAGE
+                  value: centos/postgresql-10-centos7
                 image: quay.io/3scale/3scale-operator:v0.5.0
```

Comment:
Reference to a new operator image even if code does not change? :thinking:

Reply:
The operator version is v0.5.0. That does not change.